src/tests/Microsoft.PowerFx.Interpreter.Tests.Shared/LanguageTest.cs

```diff
@@ -49,13 +49,6 @@ public void GetDefaultLanguageTest()
         [Fact]
         public void GetLanguageForNullCulture()
         {
-            var runtimeConfig = new RuntimeConfig();
-            var runner = new EvalVisitor(runtimeConfig, CancellationToken.None);
```

Comment:
This test is failing because it hits private surface area.  EvalVisitor is private, and we have to go through ParsedExpression, which will set the culture.